src/Sentry/Protocol/SpanContext.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Sentry.Protocol
+{
+    /// <summary>
+    /// Span metadata used for sampling.
+    /// </summary>
+    public class SpanContext : ISpanContext
```

Comment:
I still don't know why we need `SpanContext` here though. Both guidelines and Java has it, but it's not used in sampling.